Sources/Enums.swift

```diff
@@ -1,10 +1,9 @@
 
 public enum ParameterLocation: String, Codable {
+    case path
     case query
     case header
-    case path
-    case formData
-    case body
+    case cookie
 }
```

Comment:
#### [Parameters Location](https://swagger.io/specification/#parameterObject) in version 3.0:

There are four possible parameter locations specified by the in field:

**path** - Used together with Path Templating, where the parameter value is actually part of the operation's URL. This does not include the host or base path of the API. For example, in /items/{itemId}, the path parameter is itemId.

**query** - Parameters that are appended to the URL. For example, in /items?id=###, the query parameter is id.

**header** - Custom headers that are expected as part of the request. Note that RFC7230 states header names are case insensitive.

**cookie** - Used to pass a specific cookie value to the API.